public_html/layouts/basic/modules/Settings/WebserviceApps/resources/WebserviceApps.js

```diff
@@ -59,6 +69,29 @@ jQuery.Class('Settings_WebserviceApps_Index_Js', {}, {
 		AppConnector.request(params).then(function (data) {
 			progress.progressIndicator({'mode': 'hide'});
 			app.showModalWindow(data, function (container) {
+				const prevButton = container.find('.previewPassword');
```

Comment:
This password functionality should be universal and added in a file https://github.com/YetiForceCompany/YetiForceCRM/blob/developer/public_html/layouts/resources/Fields.js